### PR TITLE
Change qualtrics url to accomodate their change.

### PR DESCRIPTION
**Pull request by dcadams**
and fix the broken tests.

### CODE REVIEW

**Comment by dcadams** [comment]
@stvstnfrd

File: qualtricssurvey/qualtricssurvey.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
     )
     your_university = String(
         display_name=_('Your University:'),
-        default=_('stanford'),
+        default=_('stanforduniversity'),
```
Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: I think we actually _don't_ want this marked for translation, right? Isn't this like an account name?

File: qualtricssurvey/tests.py [hunk 2]
```diff
@@ -30,11 +30,11 @@ def test_student_view(self):
         fragment = xblock.student_view()
 
         url_frag = (
-            'href="https://stanford.qualtrics.com/SE/?SID=Enter '
-            'your survey ID here.&amp;a='
+            'href="https://stanforduniversity.qualtrics.com/jfe/form/Enter '
```
Comment (stvstnfrd) [inline qualtricssurvey/tests.py:33]: Nit per f2f: We may want to stick the base part of the URL here into a variable/constant.

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

File: qualtricssurvey/tests.py (hunk 2 above)
Comment (dcadams) [reply qualtricssurvey/tests.py:33]: If another test gets added I'll change it.

File: qualtricssurvey/qualtricssurvey.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ class QualtricsSurvey(StudioEditableXBlockMixin, XBlock):
     )
     your_university = String(
         display_name=_('Your University:'),
-        default=_('stanford'),
+        default= 'stanforduniversity',
```
Comment (stvstnfrd) [inline qualtricssurvey/qualtricssurvey.py]: PEP8 violation; no space between `=` and `kwargs`.

**Comment by stvstnfrd** [comment]
@dcadams Just the one PEP8 violation, otherwise lgtm.

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**